pdfminer/pdfdocument.py

```diff
@@ -977,7 +977,7 @@ def find_xref(self, parser: PDFParser) -> int:
         else:
             raise PDFNoValidXRef("Unexpected EOF")
         log.debug("xref found: pos=%r", prev)
-        assert prev is not None
+        assert prev is not None and prev.isdigit()
```

Comment:
I'm assuming this is already the first bug that you find by using fuzzing. While running the code myself I found a couple of others. 

What do you think of separating fixing errors into separate MR's. I'm hoping that we get some great statistics from ossfuzz on how many errors we fixed this way.